client/src/components/CreationMode.vue

```diff
@@ -0,0 +1,228 @@
+     * Allows us to add more sub settings in the future
+     */
+    saveTrackSubSettings(event: true | null, target: 'autoAdvanceFrame') {
+      // Copy the newTrackSettings for modification
+      const copy: NewTrackSettings = cloneDeep(this.newTrackSettings.value);
+      const modeSettings = copy.modeSettings.Track;
+      modeSettings[target] = !!event; // Modify the value
+      this.$emit('update-new-track-settings', copy);
+    },
+    saveDetectionSubSettings(event: true | null, target: 'continuous') {
+      // Copy the newTrackSettings for modification
+      const copy: NewTrackSettings = cloneDeep(this.newTrackSettings.value);
+      const modeSettings = copy.modeSettings.Detection;
+      modeSettings[target] = !!event; // Modify the value
+      this.$emit('update-new-track-settings', copy);
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="CreationMode mb-2">
+    <v-divider />
+
+    <v-subheader>Settings</v-subheader>
+    <v-card>
+      <v-row
+        align="center"
+        dense
+      >
+        <v-col
+          class="mx-2"
+          cols="3"
+        >
+          Mode:
+        </v-col>
+        <v-col>
+          <v-combobox
+            v-model="newTrackSettings.value.mode"
+            class="ml-0"
+            x-small
+            :items="modes"
+            dense
+            hide-details
+          />
+        </v-col>
+        <v-col cols="2">
+          <v-tooltip
+            open-delay="200"
+            bottom
+            max-width="200"
+          >
+            <template #activator="{ on }">
+              <v-icon
+                small
+                v-on="on"
+              >
+                mdi-help
+              </v-icon>
+            </template>
+            <span>{{ help.mode[newTrackSettings.value.mode] }}</span>
+          </v-tooltip>
+        </v-col>
+      </v-row>
+      <v-row
+        align="center"
+        dense
+      >
+        <v-col
+          class="mx-2"
+          cols="3"
+        >
+          Type:
+        </v-col>
+        <v-col>
+          <v-combobox
+            :value="newTrackSettings.value.type"
+            class="ml-0"
+            x-small
+            :items="typeList"
+            dense
+            hide-details
+            @change="saveTypeSettings($event, 'type')"
+          />
+        </v-col>
+        <v-col cols="2">
+          <v-tooltip
+            open-delay="200"
+            bottom
+          >
+            <template #activator="{ on }">
+              <v-icon
+                small
+                v-on="on"
+              >
+                mdi-help
+              </v-icon>
+            </template>
+            <span>{{ help.type }}</span>
+          </v-tooltip>
+        </v-col>
+      </v-row>
+      <v-row
+        v-if="newTrackSettings.value.mode==='Track'"
+      >
+        <v-col>
+          <v-switch
+            :input-value="
+              newTrackSettnewTrackSettings.valueings.modeSettings.Track.autoAdvanceFrame"
```

Comment:
nice

Reply:
ahh no error because the v-if was correct so it never hit that section of the template.  Must not of noticed that the switch label didn't change.  No clue how the frame toggling worked when testing though.  Definitely a good catch and a silly paste mistake by me.